install_config/install/disconnected_install.adoc

```diff
@@ -233,6 +234,15 @@ registry.access.redhat.com/jboss-webserver-3/webserver30-tomcat7-openshift:lates
 registry.access.redhat.com/jboss-webserver-3/webserver30-tomcat7-openshift:1.1
 ----
 
+. If you are using a stand-alone registry, you must pull the *registry-console*
+image.
```

Comment:
@ahardin-rh Forgot to say earlier, we should probably also say here "If you are using a stand-alone registry or plan to enable the registry console with the integrated registry". Since that uses the same image too (https://docs.openshift.com/container-platform/3.4/install_config/registry/deploy_registry_existing_clusters.html#registry-console).